filebeat/main.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	filebeat "github.com/elastic/beats/filebeat/beat"
 	"github.com/elastic/beats/libbeat/beat"
+	"os"
```

Comment:
We normally separate the internal and external exports. See https://github.com/elastic/beats/blob/master/filebeat/harvester/log.go#L7